src/openrct2/ride/rtd/water/DinghySlide.h

```diff
@@ -24,11 +25,15 @@ constexpr RideTypeDescriptor DinghySlideRTD =
             .Drawer = GetTrackPaintFunctionDinghySlide,
             .EnabledTrackPieces = { TrackGroup::straight, TrackGroup::stationEnd, TrackGroup::liftHill, TrackGroup::slope, TrackGroup::slopeSteepUp, TrackGroup::slopeSteepDown, TrackGroup::sBend, TrackGroup::curveSmall, TrackGroup::curve },
             .ExtraTrackPieces = {},
+            .icon = SPR_RIDE_CONSTRUCTION_U_SHAPED_TRACK,
+            .tooltip = STR_RIDE_CONSTRUCTION_U_SHAPED_OPEN_TRACK_TIP,
```

Comment:
Assuming they're only needed in RTDs, can we move these to RideStringIds.h instead of the bulkier StringIds.h?